vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/regionutils/RegionFactory.java

```diff
@@ -90,41 +88,14 @@ private boolean isChromosomeInRegionFilterWithNoCoordinates(String chromosome, S
         return regions;
     }
 
-    public int getMinStart(String chromosome, QueryOptions query) {
-        QueryOptions minQuery = addChromosomeSortAndLimitToQuery(chromosome, query, true);
-        return getVariantStart(minQuery);
+    public Long getMinStart(String chromosome,  List<String> studies) {
+        return variantService.findChromosomeLowestReportedCoordinate(chromosome, studies);
     }
 
-    public int getMaxStart(String chromosome, QueryOptions query) {
-        QueryOptions maxQuery = addChromosomeSortAndLimitToQuery(chromosome, query, false);
-        return getVariantStart(maxQuery);
+    public long getMaxStart(String chromosome,  List<String> studies) {
+       return variantService.findChromosomeHighestReportedCoordinate(chromosome, studies);
```

Comment:
Remove this method and call directly to the service in any place this is being used (both in this class and in VariantExporterController)